_reports/Naveenaidu.md

```diff
@@ -0,0 +1,214 @@
+---
+layout: report
+student: Naveen Naidu
+organisation: coala
+organisation_link : https://coala.io
+project: Handle Nested Programming Languages
+project_link: https://summerofcode.withgoogle.com/projects/#5842650266075136
+tarball: https://github.com/Naveenaidu/GSoCReport-2019/raw/master/naveen_naidu.tar.gz
+mentors: >
+ [Virresh Gupta](https://github.com/virresh) & [Saurav Singh](https://github.com/damngamerz) & [Marcos Gomez](https://github.com/margobra8)
+phase:
+ - Bonding : https://gitlab.com/coala/GSoC/gsoc-2019/-/milestones/24
+ - Phase 1 : https://gitlab.com/coala/GSoC/gsoc-2019/-/milestones/27
```

Comment:
Close the relevant milestones

Reply:
Closed. Sorry for this - I was under the impression that Milestones can only be closed once the PR is merged.

Reply:
> I was under the impression that Milestones can only be closed once the PR is merged

Yep, that's true. I meant the issues. I thought you had some of them finished (like the blogpost ones). I guess I messed up. Re-opening your milestones